src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,186 @@
+
+![](resources/images/data_entry_beta/limits-add-button.png) 
+
+Once you've added a minimum and maximum limit, click the *Save limits* button.
+
+If a data value already has limits set up, you can change them or remove them. To change the limits, click the *Edit limits* button. To remove the limits, click the *Delete limits* button.
+
+![](resources/images/data_entry_beta/limits-edit-button.png) 
+
+#### Historical data
+
+To learn more about a data value, you can see the last twelve values in the *History* section of the data details sidebar. [Make sure the data details sidebar is open](), then open the *History* section
+
+![](resources/images/data_entry_beta/value-history.png) 
+
+When working through a data entry form, the *History* section is closed by default to prevent too much network data from being sent for every value.
+
+#### Audit log
+
+Every data entry cell has an audit log that shows when values were changed and who changed them. To see the audit log, [make sure the data details sidebar is open](), then open the *Audit log* section.
+
+<!-- todo: add image -->
+
+When working through a data entry form, the *Audit log* section is closed by default to prevent too much network data from being sent for every value.
+
+
+### Printing
+
+To print a data entry form, click the *Options* button in the top bar. From the dropdown menu, you can choose to print the form including cell data values, or an empty form.
+
+![](resources/images/data_entry_beta/print.png) 
+
+## Working offline
+
+You don't need to be connected to the internet or DHIS2 server to enter data. Data entered into forms while you are offline is saved to your local computer. When you reconnect to the internet/server your locally saved data automatically syncs with your DHIS2 server.
+
+To work offline, you need to open the Data Entry (beta) app while you are connected to the internet so the data entry forms can be downloaded and stored on your local computer. Downloading of forms happens automatically in the background.
+
+The badge in the header bar at the top of the screen shows your connection status. If you are not connected to the internet or DHIS2 server, the badge shows *offline*. As well as the offline badge, forms cells you enter data will show the *waiting to sync* status.
+
+![](resources/images/data_entry_beta/offline.png)
+
+
+## Features not supported in the beta version
+
+The **Data Entry (beta) app** is constantly evolving and new features are being added. There are some features that the previous Data Entry app offered that are supported yet:
+- **Custom forms** don't support CSS/JS yet. Custom forms that use CSS/JS won't be loaded. We're learning more about how custom forms are used to build a sustainable, flexible solution.
```

Comment:
I think they will be loaded but without the css and js and therefore might be broken. I'm not too sure about it, but @HendrikThePendric knows as he has implemented that part

Reply:
In that case, let's drop that sentence
```suggestion
- **Custom forms** don't support CSS/JS yet. We are currently examining the different ways custom forms are used in order to build a more sustainable, flexible solution.
```